packages/frontend2/src/app/(new)/(other)/scaling/tvl/_components/table/columns.tsx

```diff
@@ -0,0 +1,157 @@
+  }),
+  columnHelper.group({
+    id: 'data',
+    header: undefined,
+    columns: [totalColumn],
+  }),
+  columnHelper.accessor('tvl', {
+    id: 'canonical',
+    header: 'Canonical',
+    cell: (ctx) => {
+      const value = ctx.getValue()
+      if (!value.breakdown) {
+        return <UpcomingBadge />
+      }
+
+      return (
+        <ValueLockedCell
+          value={value.breakdown.canonical}
+          change={value.change?.canonical}
+          tokens={[]}
+        />
+      )
+    },
+    sortUndefined: 'last',
+    meta: {
+      align: 'center',
+      tooltip:
+        'These tokens use L1 Ethereum as their main ledger and are bridged to L2 via a canonical bridge locking tokens in L1 escrow and minting on L2 an IOU representation of that token. The value is displayed together with a percentage change compared to 7D ago.',
+      headClassName: getColumnHeaderUnderline('before:bg-purple-100'),
+    },
+  }),
+  columnHelper.accessor('tvl', {
+    id: 'external',
+    header: 'External',
+    cell: (ctx) => {
+      const value = ctx.getValue()
+      if (!value.breakdown) {
+        return <UpcomingBadge />
+      }
+
+      return (
+        <ValueLockedCell
+          value={value.breakdown.external}
+          change={value.change?.external}
+          tokens={[]}
+        />
+      )
+    },
+    sortUndefined: 'last',
+    meta: {
+      align: 'center',
+      tooltip:
+        'These tokens use some external blockchain as their main ledger and are bridged to L2 via a non-canonical bridge. Tokens are locked on their native ledger and the bridge is minting on L2 an IOU representation of that token. The value is displayed together with a percentage change compared to 7D ago.',
+      headClassName: getColumnHeaderUnderline('before:bg-yellow-200'),
+    },
+  }),
+  columnHelper.accessor('tvl', {
+    id: 'native',
+    header: 'Native',
+    cell: (ctx) => {
+      const value = ctx.getValue()
+      if (!value.breakdown) {
+        return <UpcomingBadge />
+      }
+
+      return (
+        <ValueLockedCell
+          value={value.breakdown.native}
+          change={value.change?.native}
+          tokens={[]}
+        />
+      )
+    },
+    sortUndefined: 'last',
+    meta: {
+      align: 'center',
+      tooltip:
+        'These tokens are using L2 as their ledger and are minted directly on L2. Note that for some tokens (omnichain tokens) their ledger is distributed across many blockchains and they can be moved to L2 via a burn-mint bridge. The value is displayed together with a percentage change compared to 7D ago.',
+      headClassName: getColumnHeaderUnderline('before:bg-pink-100'),
+    },
+  }),
+]
+
+export function getColumnHeaderUnderline(...classNames: string[]) {
+  return cn(
+    'relative before:absolute',
+    'before:inset-x-3 before:h-0.5',
+    'before:bottom-px',
+    'before:rounded-t-full',
+    ...classNames,
+  )
+}
```

Comment:
it's already in the codebase
`packages/frontend/src/utils/table/getColumnHeaderUnderline.ts`